node/tessel-export.js

```diff
@@ -555,6 +567,29 @@ Tessel.Pin.prototype.read = function(cb) {
   return this;
 };
 
+Tessel.Pin.prototype.mode = function(pullType, cb) {
+
+  // Set a default value to 'none';
+  if (pullType === undefined) {
+    pullType = 'none';
+  }
+
+  var mode = Tessel.Pin.pullModes[pullType];
+
+  // Ensure this pin supports being pulled
+  if (!this.pullSupported) {
+    throw new Error('Internal pull resistors are not available on this pin. Please use pins 2-7.');
+  }
+
+  // Ensure a valid mode was requested
+  if (mode === undefined) {
+    throw new Error('Invalid pull type. Must be one of: \'pullup\', \'pulldown\', or \'none\'');
```

Comment:
This check should happen immediately after `var mode = Tessel.Pin.pullModes[pullType];`